package.json

```diff
@@ -6,11 +6,11 @@
   "scripts": {
     "dev-api": "pnpm --filter api dev",
     "dev-web": "pnpm --filter web dev",
-    "build": "./scripts/build",
+    "build": "turbo build",
+    "test": "turbo test",
     "lint-web": "pnpm --filter web lint",
-    "check-types": "pnpm --recursive check-types",
-    "check-prettier": "pnpm prettier --check .",
-    "format": "pnpm prettier --write .",
+    "check-format": "prettier --check .",
+    "format": "prettier --write .",
     "generate": "pnpm --filter api generate",
     "migrate": "pnpm --filter api migrate",
```

Comment:
All these other `pnpm --filter ...` commands... are there turbo variants we should be using, or still using these the way they are?

Reply:
not really since they're package specific one time rare use pnpm --filter makes more sense then making a top level turbo task unless you want a `turbo db:generate` & `turbo db:migrate`